packages/sources/src/components/form/form-groups/confirm/form-group-confirm.args.ts

```diff
@@ -0,0 +1,70 @@
+import { HandlerFunction } from '@storybook/addon-actions';
+import { ArgTypes, noControl } from '../../../../stories-helpers';
+import { v4 as uuidv4 } from 'uuid';
+
+import { FormGroupConfirm } from '../../form.models';
+
+import { Selectable } from '../../../selectable/selectable.models';
+
+export interface FormGroupConfirmArgs {
+  id: string;
+  state?: 'invalid' | 'valid';
+  required: boolean;
+  disabled: boolean;
+  errorText?: string;
+  helpText?: string;
+}
+
+export const formGroupConfirmArgTypes: ArgTypes<FormGroupConfirmArgs> = {
+  id: {
+    control: {
+      type: 'text'
+    }
+  },
+  state: {
+    options: [undefined, 'invalid', 'valid'],
+    control: {
+      type: 'select'
+    }
+  },
+  required: {
+    control: {
+      type: 'boolean'
+    }
+  },
+  disabled: {
+    control: {
+      type: 'boolean'
+    }
+  },
+  errorText: {
+    control: {
+      type: 'text'
+    }
+  },
+  helpText: {
+    control: {
+      type: 'text'
+    }
+  }
+};
+
+export function formGroupConfirmArgsMapper(a: FormGroupConfirmArgs): FormGroupConfirm<any> {
+  return {
+    group: 'confirm',
+    id: a.id,
+    required: a.required,
+    disabled: a.disabled,
+    state: a.state,
+    errorText: a.errorText,
+    helpText: a.helpText,
+    selectables: [
+      {
+        id: uuidv4(),
+        value: 'akkoord',
+        type: 'checkbox',
+        label: 'Ik ga akkoord met de voorwaarden'
```

Comment:
Deze content ook graag via Args laten lopen, zodat we vanuit Storybook Controls met variatie kunnen spelen.

Reply:
Bij radios en checkboxes hebben we het niet gedaan omdat het er meerdere zijn. Om voor meerdere radios/checkboxes de waardes via Args te laten lopen geeft complexe code die maar moeilijk met de platte Args/Controls structuur van Storybook werkt.

Nu hebben we één checkbox. De uitgangspositie is om werkvormen zo vrij mogelijk te demonstreren. Het is een kleine moeite om deze enkele checkbox aan te sluiten.

>EN hoe zou dat dan moeten?

Zodra je het model van meerdere selectables aanpast naar een enkele selectable, zal de rest op z'n plek vallen en is de waarde die bij de selectable hoort gewoon 'n extra control in de Args definitie.